grafana-plugin/e2e-tests/integrations/maintenanceMode.test.ts

```diff
@@ -81,11 +81,7 @@ test.describe('maintenance mode works', () => {
     await page.getByTestId('integration-stop-maintenance').click();
 
     // in the modal popup, confirm that we want to stop it
-    await clickButton({
-      page,
-      buttonText: 'Stop',
-      startingLocator: page.getByRole('dialog'),
-    });
```

Comment:
I believe the modal component that we use from Grafana changed the `role` attributes across various versions.. this is much more stable